homeassistant/util/yaml/loader.py

```diff
@@ -2,157 +2,29 @@
 
 from __future__ import annotations
 
-from collections.abc import Callable, Iterator
-import fnmatch
-from io import StringIO, TextIOWrapper
+from io import StringIO
 import logging
 import os
-from pathlib import Path
-from typing import Any, TextIO, overload
-
+from typing import Any, TextIO
+
+from annotatedyaml import YAMLException, YamlTypeError
+from annotatedyaml.loader import (
+    JSON_TYPE,
+    LoaderType,
+    Secrets,
+    add_constructor as add_annotatedyaml_constructor,
```

Comment:
I think we can import this without having to wrap it and add it to `__all__`

Reply:
Oh yes, indeed